tests/functional/z_cluster/test_must_gather_modular.py

```diff
@@ -0,0 +1,114 @@
+import logging
+import pytest
+
+from ocs_ci.framework.pytest_customization.marks import brown_squad
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    tier2,
+    skipif_external_mode,
+    skipif_ms_consumer,
+    skipif_hci_client,
+    hci_provider_and_client_required,
+    turquoise_squad,
+)
+from ocs_ci.ocs.must_gather.must_gather import MustGather
+from ocs_ci.ocs.must_gather import const_must_gather
+
+logger = logging.getLogger(__name__)
+
+
+@brown_squad
+class TestMustGather(ManageTest):
+    @tier2
```

Comment:
should this be in tier1?

Reply:
I dont think we need to add the test to tier1 suite. Most customers will not use the flags..